slotting/test/Test/Cardano/Slotting/EpochInfo.hs

```diff
@@ -0,0 +1,48 @@
+module Test.Cardano.Slotting.EpochInfo where
+
+import Cardano.Slotting.EpochInfo.API (EpochInfo (..))
+import Cardano.Slotting.EpochInfo.Extend (unsafeLinearExtendEpochInfo)
+import Cardano.Slotting.EpochInfo.Impl (fixedEpochInfo)
+import Cardano.Slotting.Slot (EpochNo (EpochNo), EpochSize (EpochSize), SlotNo (SlotNo))
+import Cardano.Slotting.Time (slotLengthFromSec)
+import Data.Functor.Identity (Identity)
+import Test.Tasty (TestTree, testGroup)
+import Test.Tasty.QuickCheck as QC
+  ( Arbitrary (arbitrary),
+    choose,
+    testProperty,
+    (===),
+  )
+
+baseEpochInfo :: EpochInfo Identity
+baseEpochInfo = fixedEpochInfo (EpochSize 10) (slotLengthFromSec 10)
+
+extendedEpochInfo :: SlotNo -> EpochInfo Identity
```

Comment:
Would you add a comment explaining that `extendedEpochInfo` should be `id` when applied to the result of `fixedEpochInfo`?